README.md

```diff
@@ -108,25 +108,12 @@ import (
 ...
 ```
 
-### Pinning to the `v1` releases
+### Using `v1` releases
```

Comment:
The build is failing because this line needs to be changed in the table of contents too 🙂

Reply:
Fixed